Source/JavaScriptCore/bytecode/StubInfoSummary.h

```diff
@@ -39,6 +40,7 @@ inline bool isInlineable(StubInfoSummary summary)
 {
     switch (summary) {
     case StubInfoSummary::Simple:
+    case StubInfoSummary::Megamorphic:
```

Comment:
GetByStatus can't inline this in DFG, right?

Reply:
Yes. We try using IC again to see / collect data in DFG / FTL context.